src/EA.Iws.Web/Areas/AdminExportAssessment/Controllers/MarkAsInterimController.cs

```diff
@@ -24,6 +28,8 @@ public MarkAsInterimController(IMediator mediator)
 
             var model = new MarkAsInterimViewModel(interimStatus);
 
+            model.ShowUpdateInterimStatus = Task.Run(() => authorizationService.AuthorizeActivity(typeof(UpdateInterimStatus))).Result;
```

Comment:
This method is async so you can await the authorization service call rather than using Task.Run.